src/applications/disability-benefits/all-claims/content/waiveRetirementPay.jsx

```diff
@@ -21,7 +21,7 @@ export const waiveRetirementPayDescription = (
     <table>
       <thead>
         <tr>
-          <th />
+          <td />
```

Comment:
This seems like a linting mistake?

Reply:
Agreed seems a little weird that convert a header cell to a regular cell when the rest of the row is using header cells, but it doesn't seem to change anything when running the code and still leaves the cell blank

Reply:
I don't think so, headers need labels otherwise they aren't headers. This is the little extra cell in the top left corner, I think